integrationExamples/gpt/userId_example.html

```diff
@@ -130,6 +139,25 @@
                 // },
                 userSync: {
                     userIds: [{
+                        name: "publisherProvided",
+                        params: {
+                            eids: [{
+                                source: "example.com",
+                                uids:[{
+                                    id: "value read from cookie or local storage",
+                                    ext: {
+                                        third: "third"
```

Comment:
can this ext be empty? instead of putting third: "third"

Reply:
Ok, it's just an example 
I can remove or put there more meaningful value